rules/xcodeproj.bzl

```diff
@@ -44,6 +45,17 @@ def _srcs_info_build_files(ctx):
 
     return [path]
 
+def _xcodeproj_aspect_collect_hmap_paths(deps, target, ctx):
+    hmap_paths = []
+    for dep in deps:
+        if type(dep) == "Target" and HeaderMapInfo in dep:
```

Comment:
still shouldnt need the `type` check, as the `deps` param should only be passed a list of Target objects